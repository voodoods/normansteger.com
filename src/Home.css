.Home {
  width: 80%;
  margin: auto;
}