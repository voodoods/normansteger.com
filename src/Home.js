import React, { Component } from 'react';
import Header from './header/Header'
import Blog from './blog/Blog'
import StyledHome from './Home.styled'

class Home extends Component {
  render() {
    return (
      <StyledHome className="Home">
       <Header />
       <Blog />
      </StyledHome>
    );
  }
}

export default Home;
