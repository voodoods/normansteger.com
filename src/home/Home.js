import React, { Component } from 'react'
import Claim from '../claim/Claim'
import StyledHome from './Home.styled'

class Home extends Component {
  render() {
    return (
      <StyledHome className="Home">
        <Claim />
      </StyledHome>
    )
  }
}

export default Home
