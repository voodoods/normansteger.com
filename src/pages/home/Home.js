import React, { Component } from 'react'
import PropTypes from 'prop-types'
import Claim from '../../components/claim/Claim'
import Navbar from '../../components/navbar/Navbar'
import StyledHome from './Home.styled'

class Home extends Component {
  render() {
    return (
      <StyledHome className="Home pure-u-1">
        <Navbar activePath={this.props.location.pathname} />
        <Claim />
      </StyledHome>
    )
  }
}

Home.propTypes = {
  location: PropTypes.shape({
    pathname: PropTypes.string,
  }),
}

export default Home
