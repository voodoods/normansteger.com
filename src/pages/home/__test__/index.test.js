import React from 'react'
import { shallow } from 'enzyme'
import Home from '../'

describe('<Home /> component index', () => {
  it('expects component to match snapshot', () => {
    const props = {
      location: {
        pathname: '/',
      },
    }

    const wrapper = shallow(<Home {...props} />).dive()
    expect(wrapper).toMatchSnapshot()
  })
})
