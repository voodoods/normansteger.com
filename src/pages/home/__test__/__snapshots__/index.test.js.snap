// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> component index expects component to match snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <ForwardRef
    className="Home pure-u-1"
  >
    <Navbar
      activePath="/"
    />
    <Claim />
    <div
      className="pure-u-lg-6-24 pure-u-sm-1"
    >
      <withProps(BrowseRepository) />
    </div>
  </ForwardRef>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "children": Array [
        <Navbar
          activePath="/"
        />,
        <Claim />,
        <div
          className="pure-u-lg-6-24 pure-u-sm-1"
        >
          <withProps(BrowseRepository) />
        </div>,
      ],
      "className": "Home pure-u-1",
      "forwardedComponent": Object {
        "$$typeof": Symbol(react.forward_ref),
        "attrs": Array [],
        "componentStyle": ComponentStyle {
          "componentId": "sc-ifAKCX",
          "isStatic": true,
          "rules": Array [
            "
  width: 100%;
  height: 100%;
  margin: 6em;

  @media (max-width: 768px) {
    margin: 1em;
  }
",
          ],
        },
        "displayName": "styled.div",
        "foldedComponentIds": Array [],
        "render": [Function],
        "styledComponentId": "sc-ifAKCX",
        "target": "div",
        "toString": [Function],
        "warnTooManyClasses": [Function],
        "withComponent": [Function],
      },
      "forwardedRef": null,
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "activePath": "/",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <withProps(BrowseRepository) />,
          "className": "pure-u-lg-6-24 pure-u-sm-1",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        "type": "div",
      },
    ],
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": Array [
          <Navbar
            activePath="/"
          />,
          <Claim />,
          <div
            className="pure-u-lg-6-24 pure-u-sm-1"
          >
            <withProps(BrowseRepository) />
          </div>,
        ],
        "className": "Home pure-u-1",
        "forwardedComponent": Object {
          "$$typeof": Symbol(react.forward_ref),
          "attrs": Array [],
          "componentStyle": ComponentStyle {
            "componentId": "sc-ifAKCX",
            "isStatic": true,
            "rules": Array [
              "
  width: 100%;
  height: 100%;
  margin: 6em;

  @media (max-width: 768px) {
    margin: 1em;
  }
",
            ],
          },
          "displayName": "styled.div",
          "foldedComponentIds": Array [],
          "render": [Function],
          "styledComponentId": "sc-ifAKCX",
          "target": "div",
          "toString": [Function],
          "warnTooManyClasses": [Function],
          "withComponent": [Function],
        },
        "forwardedRef": null,
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "activePath": "/",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <withProps(BrowseRepository) />,
            "className": "pure-u-lg-6-24 pure-u-sm-1",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          "type": "div",
        },
      ],
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
